auth/src/commonMain/kotlin/com/liftric/auth/jwt/IdTokenSerializer.kt

```diff
@@ -206,6 +214,8 @@ internal object CustomAttributesSerializer : KSerializer<CognitoIdTokenClaims> {
         value.cognitoUsername.let { map["cognito:username"] = json.encodeToJsonElement(String.serializer(), it) }
         value.exp.let { map["exp"] = json.encodeToJsonElement(Long.serializer(), it) }
         value.eventId.let { map["event_id"] = json.encodeToJsonElement(String.serializer(), it) }
+        value.jti?.let { map["jti"] = json.encodeToJsonElement(String.serializer(), it) }
```

Comment:
You don't need to specify any serializer at all in the whole `serialize` function: The compiler knows it from the target `CognitoIdTokenClaims` class :)

Reply:
Hence, the `@SerialName` would need to remain (cf. my comment above).

Reply:
Cause I was questioning if you need both, cause from my perspective, either specify `@SerialName` or a customer serialiser should be sufficient.

Reply:
That's unrelated to the `@SerialName`s tho. I'm pretty sure we don't net them at all, as you suspect.